examples/contrib/detection/mean_ap.py

```diff
@@ -0,0 +1,252 @@
+"""
```

Comment:
Just a comment for ourselves that we have to advance on implementing mAP from our side